dandischema/digests/dandietag.py

```diff
@@ -122,7 +122,8 @@ def get_next_part(self) -> Optional[Part]:
             return None
 
     def get_part_etag(self, p: Part) -> Optional[str]:
-        return self._md5_digests[p.number - 1].hex()
+        d = self._md5_digests[p.number - 1]
+        return d.hex() if d is not None else None
```

Comment:
this seems to also provide a functionality fix, so we do not `.hex` a None (for a not yet processed part).  I am just surprised that this function is not tests covered at all. Probably it was covered  whenever this code lived in dandi-cli (IIRC) since we do upload.  I wonder, if while at it, a dedicated unittest should be added?

Reply:
@yarikoptic
* Note that the code still also lives in dandi-cli, as it hasn't been removed yet.
* Test added.

Reply:
> Note that the code still also lives in dandi-cli, as it hasn't been removed yet.

oh -- I haven't realized that.  Could you please remove it and RF to use dandischema's (we depend on dandischema anyways)

> Test added

Thank you!